filter/envoy/envoy-wasm-filters/envoyFilter.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: http-filter
+spec:
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: SIDECAR_OUTBOUND
```

Comment:
When I had to use istio ingress to access a webapp's UI, I was not getting logs. But when changing this variable from `SIDECAR_OUTBOUND` to `GATEWAY`, I started getting sentryflow logs. Not sure if we might have to make this configurable or it's fine because it's just one change for debugging. Just something to note.

Reply:
Changed the context to `ANY` to apply at all `listeners/routes/clusters` in both `sidecars` and `gateways`.
With this, we won't face any issues.